packages/clarity-js/src/interaction/timeline.ts

```diff
@@ -16,11 +16,17 @@ export function reset(): void {
     updates = [];
 }
 
-export function track(time: number, event: Event, target:number, x: number, y: number): void {
+export function track(time: number, event: Event, target: number, x: number, y: number, region?: number): void {
```

Comment:
Should region be a number? We won't have enough information to decode number to string in this case. Something to think about here.